javascript/javascript2/week3/fastestPresser/fastestPresser.js

```diff
@@ -0,0 +1,101 @@
+let inputElement = document.getElementById("myInput");
+let startButton = document.querySelector(".start-button");
+let outputElementS = document.querySelector(".letterS");
+let outputElementL = document.querySelector(".letterL");
+
+const keypressCount = {
+    sCount: 0,
+    lCount: 0
+};
+
+let keyPressed = false; // check if any key was pressed
+
+// Add the keydown event listener when the page loads
+document.addEventListener("keydown", getKeydownCount);
+
```

Comment:
her it better to use keyup or keypress events instead, because the event will work if the user holds the key  

you can read about it here https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event